tools/autograd/gen_python_functions.py

```diff
@@ -151,6 +151,11 @@
     "fill.Scalar",  # only used by the functionalization pass
     "lift.*",
     "normal_functional",  # only used by the functionalization pas
+    "_nested_tensor_strides",  # don't want to expose this to python
+    "_nested_tensor_offsets",  # don't want to expose this to python
+    "_nested_from_buffer",  # Don't want users in python to rely on this for nt construction and view api
+    "_nested_view_from_buffer",  # View only version of _nested_from_buffer. This will force users to only use the "safe" version.
+    "_nested_view_from_buffer_copy",
```

Comment:
i'm like 90% sure that there's some other logic in the codegen to automatically hide private ops from python, but I forget where - doesn't hurt to be paranoid 😛